packages/discord.js/src/structures/Poll.js

```diff
@@ -47,6 +47,23 @@ class Poll extends Base {
   }
 
   _patch(data) {
+    /**
+     * The answers of this poll
+     * @type {Collection<number, PollAnswer|PartialPollAnswer>}
+     */
+    this.answers ??= new Collection();
```

Comment:
Why don't we move this in the constructor as `this.answers = new Collection();`? It would mean this wouldn't need to be checked or instantiated, right?

Reply:
Yes